### PR TITLE
[Doc] Add metrics for token and authentication

### PR DESCRIPTION
**Pull request by Anonymitaet**
Add docs for https://github.com/apache/pulsar/pull/9321 and https://github.com/apache/pulsar/pull/9244.

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@RobertIndie could you please help review? Thanks

**Review by Huanli-Meng — APPROVED** [review]
LGTM

File: site2/docs/reference-metrics.md [hunk 1]
```diff
@@ -400,4 +427,4 @@ All the proxy metrics are labelled with the following labels:
 | split_read_latency_per_query | Summary | Total read latency per query. |
 | split_record_deserialize_time | Summary | Time spent on deserializing message to record. For example, Avro, JSON, and so on. |
 | split_record_deserialize_time_per_query | Summary | Time spent on deserializing message to record per query. |
-| split_total_execution_time | Summary | Total execution time . |
\ No newline at end of file
+| split_total_execution_time | Summary | Total execution time . |
```
Comment (Jennifer88huang) [inline site2/docs/reference-metrics.md:430]:
```suggestion
| split_total_execution_time | Summary | The total execution time . |
```

File: site2/docs/reference-metrics.md [hunk 2]
```diff
@@ -339,6 +341,31 @@ All the managed ledger bookie client metrics are labelled with the following lab
...
 
+### Token metrics
+
+All the token metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_expired_token_count | Counter | The number of Pulsar expired tokens. |
```
Comment (Jennifer88huang) [inline site2/docs/reference-metrics.md:352]: | pulsar_expired_token_count | Counter | The number of expired tokens in Pulsar. |

File: site2/docs/reference-metrics.md [hunk 3]
```diff
@@ -339,6 +341,31 @@ All the managed ledger bookie client metrics are labelled with the following lab
...
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+- *provider_name*: `provider_name=${provider_name}`. `${provider_name}` is the class name of the authentication provider.
+- *auth_method*: `auth_method=${auth_method}`. `${auth_method}` is the authentication method of the authentication provider.
+- *reason*: `reason=${reason}`. `${reason}` is the reason for failing authentication operation. (This label is only for `pulsar_authentication_failures_count`)
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_authentication_success_count| Counter | The number of successful authentication operations. |
+| pulsar_authentication_failures_count | Counter | The number of failing authentication operations. |
```
Comment (Jennifer88huang) [inline site2/docs/reference-metrics.md:367]: failed or failing?

Comment (Anonymitaet) [reply site2/docs/reference-metrics.md:367]: failing operation

**Review by RobertIndie — APPROVED** [review]
LGTM.

**Pull request merged**